<?xml version="1.0"?>
<launch>

<group ns="/Vehicle">
    <node name="VehicleSim" pkg="tod_vehicle_sim" type="VehicleSimNode" output="screen">
        <remap from="primary_control_cmd" to="/Vehicle/VehicleBridge/primary_control_cmd"/>
        <remap from="secondary_control_cmd" to="/Vehicle/CommandCreation/secondary_control_cmd"/>
        <remap from="odometry" to="/Vehicle/VehicleBridge/odometry"/>
        <remap from="vehicle_data" to="/Vehicle/VehicleBridge/vehicle_data"/>
    </node>
</group>
</launch>
